src/qofonoutils_p.h

```diff
@@ -18,6 +17,14 @@
 
 #include "qofono_global.h"
 
+#if (QT_VERSION >= QT_VERSION_CHECK(6,0,0))
+#  define QOfonoSkipEmptyParts Qt::SkipEmptyParts
+#  define qOfonoSort(what) std::sort((what).begin(), (what).end())
+#else
+#  define QOfonoSkipEmptyParts QString::SkipEmptyParts
+#  define qOfonoSort(what) qSort(what)
```

Comment:
Late to comment here but qSort has been deprecated years and years in Qt5 already. Nothing should be using it anymore.

Reply:
Even then `qOfonoSort` still makes sense is convenience wrapper, but yeah - it could probably be defined unconditionally.